pkgs/development/python-modules/pytest-annotate/default.nix

```diff
@@ -14,9 +14,17 @@ buildPythonPackage rec {
     sha256 = "03e4dece2d1aa91666034f1b2e8bb7a7b8c6be11baf3cf2929b26eea5c6e86f3";
   };
 
-  propagatedBuildInputs = [ pyannotate pytest ];
+  propagatedBuildInputs = [
+    pyannotate
+    pytest
```

Comment:
I've been suggesting making this a `buildInputs` so we can allow pytest version mixing.

##### not sure what you think of that

Reply:
Correct. We should document this somewhere.

Reply:
Fixed.